File: app/src/main/java/com/kfnoon/huanm/aribbble/model/Shot.java
package com.kfnoon.huanm.aribbble.model;

import java.io.Serializable;

public class Shot implements Serializable{
    public int id;
    public Images images;
    public String title;
    public int views_count;
    public int comments_count;
    public int likes_count;
    public Boolean animated;
    public String[] tags;
    public User user;
    
    public static class Images{
        public String hidpi;
        public String normal;
        public String teaser;
    }
}

File: app/src/main/java/com/kfnoon/huanm/aribbble/model/Shot.kt
package com.kfnoon.huanm.aribbble.model

data class Images(
        var hidpi: String,
        var normal: String,
        var teaser: String
)

data class Shot(
        var id: Int,
        var images: Images,
        var title: String,
        var views_count: Int,
        var comments_count: Int,
        var likes_count: Int,
        var animated: Boolean,
        var tags: List<String>,
        var user: User,
        var description: String
)